whiterabbit/src/main/java/org/ohdsi/whiteRabbit/ResourceType.java

```diff
@@ -0,0 +1,29 @@
+package org.ohdsi.whiteRabbit;
+
+import java.util.ArrayList;
+
+public enum ResourceType {
```

Comment:
Additional possible methods:
```java
/**
* Compute the resource type from the label. This makes future comparisons more robust.
* @return ResourceType associated to label or null if not found.
*/
static ResourceType fromLabel(String value);

/** Whether the ResourceType is a database. */
boolean isDatabase();
```
The latter can be done either by a constructor property, or a match `this != ResourceType.DELIMITED`.

Reply:
Thanks, I will refer back to this in a future refactoring.